### DISCUSSION TITLE
Datasource provisioning

### DISCUSSION THREAD

**Discussion by forDeFan** [grafana/grafana-infinity-datasource#573] [Q&A] [answered]
Hi guys,

Trying to configure yaml for Infinity provisioning, unfortunately that info missing on plugin page.
Anybody done that already and can share set up in yaml ?

**Comment by yesoreyeram** [comment, marked as answer]
You can get the details from the config page itself.. setup a working version manually and in the config page search for provisioning and you will get the script you want.

**Reply by yesoreyeram** [reply]
What info missing on plugin page? Did you search for provisioning in the plugin page?

**Comment by forDeFan** [comment]
Yes indeed - when was visiting provisioning tab scripts wasn't visible, today they are. 
Thank you @yesoreyeram for your effort.

For next visitor on the topic, here's direct link to plugin provisioning scripts: https://sriramajeyam.com/grafana-infinity-datasource/wiki/provisioning
